lib/fog/compute/terremark.rb

```diff
@@ -1,10 +1,10 @@
 module Fog
   module Compute
     class Terremark < Fog::Service
-      autoload :Common, 'fog/compute/terremark/common'
-      autoload :Parser, 'fog/compute/terremark/parser'
-      autoload :Real, 'fog/compute/terremark/real'
-      autoload :Mock, 'fog/compute/terremark/mock'
+      autoload :Common, File.expand_path('../terremark/common', __FILE__)
+      autoload :Parser, File.expand_path('../terremark/parser', __FILE__)
+      autoload :Real, File.expand_path('../terremark/real', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.